RestApiForGroup2/client.js

```diff
@@ -0,0 +1,99 @@
+var request = require('request');
+request({
+	url: 'http://127.0.0.1:3000/entities',
```

Comment:
The Base URL should be in a const ("http://127.0.0.1:3000/") or even a nicer configuration file that uses json. Checkout: https://stackoverflow.com/questions/5869216/how-to-store-node-js-deployment-settings-configuration-files